crates/matrix-sdk-ui/src/room_list_service/state.rs

```diff
@@ -97,9 +122,12 @@ impl StateMachine {
             }
 
             Running => {
-                // We haven't sync for a while so we should go back to recovering
-                if self.last_sync_date.elapsed() > self.delay_before_recover {
+                // We haven't changed the state for a while, we go back to `Recovering` to avoid
```

Comment:
Let's not talk about _sync_ but rather about _state update_ here. A _sync_ happens in `RoomListService`, not here.